fili-core/src/main/java/com/yahoo/bard/webservice/web/TablesApiRequest.java

```diff
@@ -222,6 +225,43 @@ public TablesApiRequest(
     }
 
     /**
+     * All argument constructor  for withers.
+     * @param format response data format JSON or CSV. Default is JSON.
+     * @param paginationParameters The parameters used to describe pagination.
+     * @param uriInfo The URI of the request object.
+     * @param builder The response builder for this request.
+     * @param tables set of logical tables.
+     * @param table logical table.
+     * @param granularity Requested time granularity.
+     * @param dimensions Grouping dimensions / Dimension constraint
+     * @param metrics Metrics constraint
+     * @param intervals Data / Time constraint
+     * @param filters Filter constraint
+     */
+    private TablesApiRequest(
```

Comment:
Why are we making this `private`?
Why do we want a all-args constructor?
It would be nice to explain in change log